metaci/templates/layout_full.html

```diff
@@ -87,7 +87,7 @@ <h3 class="slds-text-heading--medium">Create Org</h3>
         </div>
         </a>
       </li>
-      {% if request.user.is_superuser %}
+      {% if request.user.is_staff %}
```

Comment:
Not still superuser?

Reply:
No because is_superuser isn't really right for how Django's admin access works.  The purpose of is_staff is to control if you have access to admin.  If you only have is_staff and no perms, you can get to admin but not see anything.  In our migration scripts, we remove is_staff from users after adding them to groups that grant what is_staff used to grant on the front end.  This config allows you to grant users is_staff and, for example, the ability to edit only Orgs.

Reply:
Ah, yep.